Wrappers/Python/test/test_BlockDataContainer.py

```diff
@@ -959,3 +959,79 @@ def assertDataIsTheSame(self, data, idxs, msg=None):
                 k += 1
 
         assert wrong == 0
+
+class TestBlockDataContainerGeometry(BDCUnittest):
+    def test_allocate(self):
```

Comment:
Is this testing allocate for the BlockDataContainer or the ImageDataContainers?

Reply:
Thanks for adding the unit test